clientv3/integration/watch_test.go

```diff
@@ -50,6 +50,10 @@ func runWatchTest(t *testing.T, f watcherTest) {
 	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 3})
 	defer clus.Terminate(t)
 
+	if len(clus.Members) != 3 {
```

Comment:
I've tried stepping through this test and there is check [here](https://github.com/etcd-io/etcd/blob/9bb134233a42c0339cd7c5355dd9c2772e580944/tests/framework/integration/cluster.go#L232) 
that waits for all members to be up. I think it checks for size implicitly.

Do we really need this extra check?

Reply:
Right, I don't think we should add the additional check either.